addon/components/password-toggle.js

```diff
@@ -2,20 +2,17 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
     didInsertElement: function() {
-        $('button').on('click', function(){
-          var text = $(this).text();
-          var $parent = $(this).parent();
-          var $input = $('input', $parent).clone();
+        $('button').on('click', function() {
+            var text = $(this).text();
+            var $input = $(this).parent().find('input');
 
-          $('input', $parent).remove();
```

Comment:
We should be good! Just did some digging, and looks like IE9+ supports it. Thanks guys!